libraries/AP_Common/AP_Common.h

```diff
@@ -23,7 +23,10 @@
 #include <AP_HAL/AP_HAL_Boards.h>
 #include <stdint.h>
 #include <stdlib.h>
+
+#if !defined(__ANDROID__) && !defined(__BORLANDC__)
```

Comment:
I'd prefer to see HAVE_STDBOOL_H checks, and make waf configured check for include of stdbool.h